src/main/java/de/tum/in/www1/artemis/web/rest/ExerciseResource.java

```diff
@@ -289,7 +289,7 @@ private StatsForInstructorDashboardDTO populateCommonStatistics(Exercise exercis
             return forbidden();
         exerciseService.cleanup(id, deleteRepositories);
         log.info("Cleanup build plans was successful for Exercise : {}", id);
-        return ResponseEntity.ok().headers(HeaderUtil.createAlert(applicationName, "Cleanup was successful. Repositories have been deleted: " + deleteRepositories, "")).build();
+        return ResponseEntity.ok().build();
```

Comment:
why did you change this and remove the feedback?

Reply:
Because I show the alerts now on the client after I received successful response.